<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VisitorChatView</class>
 <widget class="QWidget" name="VisitorChatView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>999</width>
    <height>594</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QSplitter" name="splitter_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QGridLayout" name="gridLayout_3">
        <property name="leftMargin">
         <number>8</number>
        </property>
        <item row="1" column="0" colspan="2">
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <property name="spacing">
             <number>0</number>
            </property>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Expanding</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>0</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="cmdSeize">
              <property name="text">
               <string>Seize</string>
              </property>
              <property name="icon">
               <iconset resource="Images.qrc">
                <normaloff>:/Images/Chat/seize_visitor.png</normaloff>:/Images/Chat/seize_visitor.png</iconset>
              </property>
              <property name="flat">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="cmdRedirect">
              <property name="text">
               <string>Redirect</string>
              </property>
              <property name="icon">
               <iconset resource="Images.qrc">
                <normaloff>:/Images/d.png</normaloff>:/Images/d.png</iconset>
              </property>
              <property name="flat">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="MessageEditor" name="txtMessage">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="sizeIncrement">
               <size>
                <width>1</width>
                <height>1</height>
               </size>
              </property>
              <property name="baseSize">
               <size>
                <width>10</width>
                <height>50</height>
               </size>
              </property>
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Sunken</enum>
              </property>
              <property name="lineWidth">
               <number>10</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="cmdSend">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="focusPolicy">
               <enum>Qt::NoFocus</enum>
              </property>
              <property name="text">
               <string>Send</string>
              </property>
              <property name="autoDefault">
               <bool>true</bool>
              </property>
              <property name="default">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item row="0" column="0" colspan="2">
         <widget class="ChatControl" name="chat" native="true"/>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="widget312" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>1</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="QTabWidget" name="tabDetails">
         <property name="tabPosition">
          <enum>QTabWidget::South</enum>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="VisitorDetailsView" name="Details">
          <attribute name="title">
           <string>Details</string>
          </attribute>
         </widget>
         <widget class="CobrowseView" name="Cobrowse">
          <attribute name="title">
           <string>Cobrowse</string>
          </attribute>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MessageEditor</class>
   <extends>QPlainTextEdit</extends>
   <header>messageeditor.h</header>
  </customwidget>
  <customwidget>
   <class>CobrowseView</class>
   <extends>QWidget</extends>
   <header>cobrowseview.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>VisitorDetailsView</class>
   <extends>QWidget</extends>
   <header>visitordetailsview.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ChatControl</class>
   <extends>QWidget</extends>
   <header>chatcontrol.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="Images.qrc"/>
 </resources>
 <connections/>
</ui>
